Makefile

```diff
@@ -29,3 +22,9 @@ clean:
 	rm -rf build
 
 .PHONY: all lint test race msan tools clean build
+
+build-simd-docker:
+	docker build -t jackzampolin/simd:v0.42.3 -f ./docker/simd/Dockerfile ./docker/simd/
```

Comment:
TODO: make version a variable at the top of makefile and keep up to date. Alternately pull in data from go.mod and make that the authoritative SDK version to run against:  `go list -m -u -f '{{.Version}}' github.com/cosmos/cosmos-sdk`. If taking second option please make sure that the docker files get built with the right arg and it gets used in the tests correctly